src/single_random_draw.rs

```diff
@@ -171,32 +145,36 @@ mod tests {
 
     #[test]
     fn select_coins_skip_negative_effective_value() {
-        let target: Amount = Amount::from_str("1 cBTC").unwrap() - CHANGE_LOWER;
+        let target: Amount = Amount::from_str("2 cBTC").unwrap() - CHANGE_LOWER;
 
-        let mut weighted_utxos: Vec<WeightedUtxo> = vec![WeightedUtxo {
+        let mut weighted_utxos = create_weighted_utxos();
+        let mut negative_eff_value = vec![WeightedUtxo {
             satisfaction_weight: Weight::ZERO,
             utxo: TxOut {
                 value: Amount::from_str("1 sat").unwrap(),
                 script_pubkey: ScriptBuf::new(),
             },
         }];
 
-        let result = select_coins_srd(target, FEE_RATE, &mut weighted_utxos, &mut get_rng())
+        weighted_utxos.append(&mut negative_eff_value);
```

Comment:
why do you create a vector instead of just pushing the additional value?

Reply:
Thanks.  I agree it's sloppy to append and we can just use push instead.  Updated now.